week_1/Project/youtube-fe/styles/video.css

```diff
@@ -0,0 +1,56 @@
+.video-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
@codal-kcolaco what is fr here?

Reply:
@abhi-codal fr stands for fraction and the above code defines four columns with an equal width. Each column will take equal fraction of space in grid.